src/api/stats/common/getApyBreakdown.ts

```diff
@@ -0,0 +1,51 @@
+import BigNumber from "bignumber.js";
+
+import getFarmWithTradingFeesApy from '../../../utils/getFarmWithTradingFeesApy';
+import { compound } from '../../../utils/compound';
+
+import { BASE_HPY } from '../../../constants';
+
+const beefyPerformanceFee = 0.045;
```

Comment:
It was discussed in development https://discord.com/channels/755231190134554696/758368074968858645/855756241715462154  that the performance fee can change, so if that’s true in practice can we parameterise that please